### ISSUE TITLE
Move from Brace Forms to Formspree.io

### ISSUE THREAD

**Issue by jasonkliu** [jasonkliu/demos#8]
tldr: Your form at `www.yale.edu/demos/` (and `jasonkliu.github.io/demos/`) will stop working unless you take action soon. Please migrate to formspree.io or your own server before January 19th.

---

Brace.io is closing down, and Brace Forms has been open sourced. You have the option to move to formspree.io, a new service based on the open sourced code and hosted by Assembly.com. Check it out at https://assembly.com/formspree. Alternatively you can host your forms using your own server.

WHAT YOU SHOULD DO:

You must change your form URL to formspree.io, or your own hosting, and re-confirm by January 19th. To do this:
1. Change the action URL on your form from `http://forms.brace.io/your%40email.com` to `http://formspree.io/your@email.com`, or use your own server. Instructions for self hosting can be found at https://github.com/braceio/forms.
2. Submit the form once, and click "confirm" in the generated confirmation email.

GET INVOLVED!

The future of formspree is in the hands of its community. If you're a developer, designer or doer of things, get involved at `https://assembly.com/formspree`. By participating, you're helping to ensure that formspree will continue to thrive.

---

You are receiving this email because you have a form on Brace Forms. If you have questions please email us at team@brace.io.
